GUI/UI/UIStatusBar.cpp

```diff
@@ -30,11 +32,28 @@ void GameUI_StatusBar_OnEvent_Internal(const String &str, unsigned int ms) {
     game_ui_status_bar_event_string_time_left = OS_GetTime() + ms;
 }
 
-void GameUI_StatusBar_OnEvent(const String &str, unsigned int num_seconds) {
-    GameUI_StatusBar_OnEvent_Internal(str, 1000 * num_seconds);
+void GameUI_SetStatusBar(const String &str) {
```

Comment:
Was the time override used much/ at all?

Reply:
the only place it was used is https://github.com/gp-alex/world-of-might-and-magic/blob/master/src/Application/Game.cpp#L1560

and i assume that's some debug script that there's no way to call now even

Reply:
i am also going to add GameUI_SetStatusBar(int lstr_code, ...) to mimic GameUI_SetStatusBar(localize->FormatString()) because the code is just riddled with that